app/grandchallenge/algorithms/templates/algorithms/data_tables/job_list.html

```diff
@@ -26,7 +26,7 @@
    data-title="Result Info"
    data-output="
    {% if job.status == job.SUCCESS %}
-        {% for output in job.outputs.all %}{% if output.value %}{{ output.value }}{% endif %}{% empty %}No output produced.{% endfor %}
+        {% if job.rendered_result_text %}{{ job.rendered_result_text|force_escape }}{% else %}No output produced.{% endif %}
```

Comment:
For my understanding: why `force_escape` over `escape`?

Reply:
Changed to json_script